bin/oref0-setup.sh

```diff
@@ -169,6 +173,19 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
       # Force uppercase, just in case the user entered ww
       radio_locale=${radio_locale^^}
 
+      # check if user has a TI USB stick and a WorldWide pump and want's to reset the USB subsystem during mmtune if the TI USB fails
+      ti_usb_ww0="no" # assume you don't want it by default
+      ti_usb_ww1=""
```

Comment:
Should we rename this to something more descriptive, like ti_ww_reset or something?